appleseed-Installer/PathSelectDialog.wxs

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--//**********************************************************************/
+// Copyright (c) 2021 appleseed.
+// All rights reserved.
+// 
+//**************************************************************************/
+// DESCRIPTION: Contains the Dll Entry stuff
+// AUTHOR: Omid Ghotbi (TAO) omid.ghotbi@gmail.com www.3dfine.com
+//***************************************************************************/-->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <UI Id="PathSelectDialogUI">
+
+      <DialogRef Id="ProgressDialog"/>
+      
+      <Property Id="Install">installbtn</Property>
+      <Binary Id="installbtn" SourceFile="Resources/btInstall.png"/>
+
+      <Dialog Id="PathSelectDialog" Width="555" Height="416" Title="appleseed : Path Selection">
+
+        <Control Id="background" Type="Bitmap" Text="bgImage" Width="555" Height="416" X="0" Y="0" TabSkip="no" />
+
+        <Control Id="explanationText" X="230" Y="100" Width="300" Height="100" NoWrap="no" RightAligned="no" Transparent="yes" Type="Text" Text="{\TahomaNormal}Setup should automatically find the path to 3dsmax, but if it did not you must point it to the root of 3dsmax folder. " />
```

Comment:
3dsmax -> Autodesk® 3ds Max®

At least for the first mention; the second one can probably remain "3ds Max".